core/src/banking_stage.rs

```diff
@@ -2019,25 +2017,33 @@ impl BankingStage {
             id,
         );
 
-        if let Some(new_sigverify_stats) = &new_tracer_stats_option {
-            tracer_packet_stats.aggregate_sigverify_tracer_packet_stats(new_sigverify_stats);
-        }
-
-        // Track all the packets incoming from sigverify, both valid and invalid
-        slot_metrics_tracker.increment_total_new_valid_packets(passed_sigverify_count);
-        slot_metrics_tracker.increment_newly_failed_sigverify_count(failed_sigverify_count);
-
         let mut dropped_packets_count = 0;
         let mut newly_buffered_packets_count = 0;
-        Self::push_unprocessed(
-            buffered_packet_batches,
+        for ReceivePacketResults {
```

Comment:
nit: Maybe rename `ReceivePacketResults` to something more suggestive that it came from the deserialize stage, maybe `DeserializedPacketResults`?

Reply:
Yeah, probably best to rename this, as was pointed out in another PR having "Result" in the name can be confusing since it's not a rust Result